android/app/src/main/cpp/jitter_buffer.cpp

```
@@ -0,0 +1,148 @@
+        // counter. We grow conservatively (+1) rather than jumping to max —
+        // the link may have just been transiently bad.
+        if (targetDepth_ < audio_config::kJitterMaxDepth) {
+            ++targetDepth_;
+        }
+        stableIntervalsCount_ = 0;
+    } else {
+        // Underrun-free interval. Count it; shrink only after the link has
+        // proved itself across several intervals so a brief calm doesn't
+        // shrink us right back into the danger zone.
+        ++stableIntervalsCount_;
+        const size_t shrinkAfter =
```

Clang (14.0.6)

[warning] variable 'shrinkAfter' is not initialized

(cppcoreguidelines-init-variables)